Makefile.build

```diff
@@ -765,6 +765,21 @@ $(VDFILE).d: $(D_DEPEND_BEFORE_SRC) $(VFILES) $(D_DEPEND_AFTER_SRC) $(COQDEPBOOT
 	$(SHOW)'COQDEP    VFILES'
 	$(HIDE)$(COQDEPBOOT) -boot $(DYNDEP) $(VFILES) $(TOTARGET)
 
+###########################################################################
+# Bisect Coverage
+###########################################################################
```

Comment:
Wouldn't this fit `Makefile.dev` better?

Reply:
Maybe, but I always get pain with the `test-suite` target not being in scope.